From bca251aebc0190bd3eb62ca35fbb8d014d84421c Mon Sep 17 00:00:00 2001
From: "Kevin F. Haggerty" <haggertk@lineageos.org>
Date: Sat, 12 Aug 2023 07:52:25 -0600
Subject: [PATCH] Bump Security String to 2023-08-05

Implemented:
============
CVE:            References:    Type:  Severity:  Updated AOSP versions:
CVE-2023-21265  A-262521447    ID     High       11, 12, 12L, 13
CVE-2023-21267  A-218495634    ID     High       11, 12, 12L, 13
CVE-2023-21268  A-264880895    DoS    High       11, 12, 12L, 13
CVE-2023-21281  A-265431505    EoP    High       11, 12, 12L, 13
CVE-2023-21282  A-279766766    RCE    Critical   11, 12, 12L, 13
CVE-2023-21283  A-280797684    ID     High       11, 12, 12L, 13
CVE-2023-21284  A-260729089    DoS    High       11, 12, 12L, 13
CVE-2023-21285  A-271851153    ID     High       11, 12, 12L, 13
CVE-2023-21286  A-277740082    EoP    High       11, 12, 12L, 13
CVE-2023-21287  A-278221085    RCE    High       11, 12, 12L, 13
CVE-2023-21288  A-276294099    ID     High       11, 12, 12L, 13
CVE-2023-21289  A-272020068    ID     High       11, 12, 12L, 13
CVE-2023-21290  A-264880689    DoS    High       11, 12, 12L, 13
CVE-2023-21292  A-236688380    ID     High       11, 12, 12L, 13

Previously Implemented:
=======================
CVE:            References:    Type:  Severity:  Updated AOSP versions:  Prior Change:
CVE-2023-21272  A-227471459    EoP    High       11, 12, 12L             2f24979c13a3
CVE-2023-21273  A-272783039    RCE    Critical   11, 12, 12L, 13         433f913eb4

Not Implemented:
=======================
None

Not Applicable (platform source):
=================================
CVE:            References:    Type:  Severity:  Updated AOSP versions:
CVE-2023-20965  A-250574778    EoP    High       13
CVE-2023-21132  A-253043218    EoP    High       12, 12L, 13
CVE-2023-21133  A-253043502    EoP    High       12, 12L, 13
CVE-2023-21134  A-253043495    EoP    High       12, 12L, 13
CVE-2023-21140  A-253043490    EoP    High       12, 12L, 13
CVE-2023-21242  A-277824547    EoP    High       13
CVE-2023-21269  A-271576718    EoP    High       13
CVE-2023-21270  A-283006437    EoP    High       12, 12L, 13
CVE-2023-21271  A-269455813    ID     High       12, 12L, 13
CVE-2023-21274  A-269456018    ID     High       12, 12L, 13
CVE-2023-21275  A-278691965    EoP    High       12, 12L, 13
CVE-2023-21276  A-213170822    ID     High       12, 12L, 13
CVE-2023-21277  A-281018094    ID     High       12, 12L, 13
CVE-2023-21278  A-281807669    EoP    High       12, 12L, 13
CVE-2023-21279  A-277741109    ID     High       12, 12L, 13
CVE-2023-21280  A-270049379    DoS    High       12, 12L, 13

Change-Id: I77a82e2673238f726352eb562f6deca2f4cb78b2
---
 core/version_defaults.mk | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/core/version_defaults.mk b/core/version_defaults.mk
index 183a223f2b..1e00b751d1 100644
--- a/core/version_defaults.mk
+++ b/core/version_defaults.mk
@@ -250,7 +250,7 @@ ifndef PLATFORM_SECURITY_PATCH
     #  It must be of the form "YYYY-MM-DD" on production devices.
     #  It must match one of the Android Security Patch Level strings of the Public Security Bulletins.
     #  If there is no $PLATFORM_SECURITY_PATCH set, keep it empty.
-      PLATFORM_SECURITY_PATCH := 2023-07-05
+      PLATFORM_SECURITY_PATCH := 2023-08-05
 endif
 .KATI_READONLY := PLATFORM_SECURITY_PATCH
 
-- 
2.34.1

